src/modules/app/dtos/get-health-check.ts

```diff
@@ -0,0 +1,6 @@
+import { Expose } from 'class-transformer';
+
+export class AppGetHealthOutput {
```

Comment:
AppGetHealthOutput -> AppGetHealthDto or AppGetHealthOutputDto
also, it's good to have extention like AppGetHealth.dto.ts or AppGetHealth.output.dto.ts e.c